graylog2-web-interface/src/views/components/visualizations/GenericPlot.tsx

```diff
@@ -174,17 +174,15 @@ const usePlotLayout = (layout: Partial<Layout>) => {
   }, [colors, interactive, layout, theme]);
 };
 
-const usePlotChatData = (chartData: Array<any>, setChartColor: (data: ChartConfig, color: ColorMapper) => ChartColor) => {
+const usePlotChartData = (chartData: Array<any>, setChartColor: (data: ChartConfig, color: ColorMapper) => ChartColor) => {
   const theme = useTheme();
   const { colors } = useContext(ChartColorContext);
 
   return useMemo(() => chartData.map((chart) => {
     if (setChartColor && colors) {
       const conf = setChartColor(chart, colors);
 
-      if (chart.type === 'pie') {
```

Comment:
Initially I added the `bar` chart type here as well, but I later removed this check completely. This way we do not have to remember making this change when adding charts in the future. In addition this layout setting should not have an effect on charts which do not need this styling.